src/analysis/inlay_hints.mli

```diff
@@ -0,0 +1,12 @@
+(** Builds the list of inlay hints to be displayed on a document. *)
+
+type hint = Lexing.position * string
+
+val run :
+  hint_let_binding:bool
+  -> hint_pattern_binding:bool
+  -> avoid_ghost_location:bool
+  -> start:Lexing.position
+  -> stop:Lexing.position
+  -> Mpipeline.t
```

Comment:
```suggestion
  -> Mtyper.typedtree
```

Reply:
I go for `Typedtree.structure` instead of the `Mtyper.typedtree`, is that ok for you?

Reply:
It makes sens if the feature only works for implementations.

Reply:
Yes